hypervisor/src/kvm/mod.rs

```diff
@@ -101,6 +103,18 @@ fn vec_with_array_field<T: Default, F>(count: usize) -> Vec<T> {
     vec_with_size_in_bytes(vec_size_bytes)
 }
 
+impl KvmVm {
+    ///
+    /// Creates an emulated device in the kernel, return the device FD.
+    ///
+    /// See the documentation for `KVM_CREATE_DEVICE`.
+    fn create_device_fd(&self, device: &mut CreateDevice) -> vm::Result<DeviceFd> {
+        self.fd
+            .create_device(device)
+            .map_err(|e| vm::HypervisorVmError::CreateDevice(e.into()))
+    }
+}
+
```

Comment:
I don't really see the benefit of adding this abstraction layer. Can we just call `self.fd.create_device()` from the `create_device` `Vm` method?

Reply:
Agree. It doesn't save much code. I changed it back.